mule-extensions-api/src/main/java/org/mule/runtime/extension/api/ExtensionConstants.java

```diff
@@ -293,5 +293,20 @@
    */
   public static final String OBJECT_STORE_ELEMENT_NAMESPACE = "os";
 
+  /**
+   * The name of the parameter which sets the item limit per poll for polling sources.
+   *
+   * @since 1.4.0
+   */
+  public static final String POLLING_SOURCE_LIMIT_PARAMETER_NAME = "maxItemsPerPoll";
+
+  /**
+   * The description of the parameter which sets the item limit per poll for polling sources.
+   *
+   * @since 1.4.0
+   */
+  public static final String POLLING_SOURCE_LIMIT_PARAMETER_DESCRIPTION = "Limits the amount of items per poll that " +
```

Comment:
this description is user facing, and a user doesn't know what a polling source is. I would leave this as just `Limits the amount of items per poll`